tests/net/context/src/main.c

```diff
@@ -380,24 +290,19 @@ static bool net_ctx_connect_v6(void)
 				  sizeof(struct sockaddr_in6),
 				  connect_cb, 0, INT_TO_POINTER(AF_INET6));
 	if (ret || cb_failure) {
-		TC_ERROR("Context connect IPv6 UDP test failed (%d)\n", ret);
-		return false;
+		zassert_true(0, "Context connect IPv6 UDP test failed");
```

Comment:
Please change this so that "ret || cb_failure" in inside zassert like you have in other places of the code.

Reply:
Done